tools/buildimg.sh

```diff
@@ -9,8 +9,9 @@ VERSION=$(grep -E '^(version|tool\.poetry\.version) *= *"[^"]+"' "$PYPROJECT_TOM
 echo $VERSION
 
 # build distribution
-uv build
+#uv build
```

Comment:
why comment this out?

Reply:
I should not check in this file.  I take it out.  Thanks!